lib/practice_exercises.rb

```diff
@@ -1,13 +1,46 @@
-
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
```

Comment:
because `delete_at` takes O(n) time complexity, your total method has a time complexity of O(n<sup>2</sup>)